random_chat/conanfile.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class TelegramWordnet(ConanFile):
+    settings = "os", "compiler", "build_type", "arch", "cppstd"
+    generators = "cmake"
+
+    def requirements(self):
+        self.requires("tgbot_cpp/1.1@jgsogo/stable")
```

Comment:
I would add the boost dependency explicitly too

Reply:
nope, let Conan resolve transitive dependencies...